packages/darklang/languageTools/writtenTypes.dark

```diff
@@ -238,4 +240,5 @@ module Darklang =
 
 
       type ParsedFile =
-        | PackageFunctions of SourceRange * fns: List<PackageFn.PackageFn>
\ No newline at end of file
+        | PackageFunctions of SourceRange * fns: List<PackageFn.PackageFn>
+        | PackageTypes of SourceRange * types: List<PackageType>
```

Comment:
Right now, this supports either a list of package types, or a list of package functions.
I think this might better represent the intention:

```fsharp
type PackageItem = 
  | Function of PackageFn.PackageFn
  | Type of PackageType

type ParsedFile = 
  | PackageItems of SOurceRange * items: List<PackageItem>
```

Reply:
at least, I think that better matches the `choice` in `grammar.js`